libutils/json-utils.c

```diff
@@ -297,6 +306,15 @@ bool JsonParseCsvFile(const char *input_path, size_t size_max, JsonElement **jso
         return false;
     }
 
+    if (JsonLength(json) == 0)
+    {
+        Log(LOG_LEVEL_WARNING,
+            "%s: CSV file '%s' is not empty, but nothing was parsed",
+            myname);
+        Log(LOG_LEVEL_WARNING,
+            "Make sure the file contains DOS (CRLF) line endings");
```

Comment:
Could we make it this only show up if there are newlines in the file? Something like `fseek(fin, 0); if (contains_newline(fin))...`.